Classes/View Controllers/UIViewController+Safari.swift

```diff
@@ -12,8 +12,12 @@ import SafariServices
 extension UIViewController {
 
     func presentSafari(url: URL) {
-		guard let safariViewController = try? SFSafariViewController.configured(with: url) else { return }
-        route_present(to: safariViewController)
+        if UserDefaults.standard.shouldOpenExternalLinksInSafari {
+            UIApplication.shared.open(url)
+        } else {
+            guard let safariViewController = try? SFSafariViewController.configured(with: url) else { return }
```

Comment:
What about putting this in a `do/catch` and asserting if this fails?

Reply:
Good suggestion!